schema-loader/src/main/java/com/scalar/db/schemaloader/SchemaOperator.java

```diff
@@ -413,9 +414,9 @@ public void importTables(List<ImportTableSchema> tableSchemaList) throws SchemaL
       String table = tableSchema.getTable();
       try {
         if (tableSchema.isTransactionTable()) {
-          transactionAdmin.get().importTable(namespace, table);
+          transactionAdmin.get().importTable(namespace, table, Collections.emptyMap());
```

Comment:
I think we should add `Map<String, String> options` to `ImportTableSchema`, and pass it to `importTable()` as the third argument. What do you think?

Reply:
Ok. 
Similarly to `TableSchema`, we could define custom options for each table in the schema file **or** pass options globally via CLI argument that will be applied to each table. 
I realized that setting a given option via the CLI **and** also in the schema file will throw an exception because of [this](https://github.com/scalar-labs/scalardb/blob/ee0e9a26f5afcebc532f9d19f3d7afe6308133c5/schema-loader/src/main/java/com/scalar/db/schemaloader/TableSchema.java#L159) (the same option key will be added twice in the list builder and this is not allowed by default). I don't know if this behavior is intended eventually. If an option is set globally as CLI argument and in the schema file, using the value specified in the schema file is more user-friendly in my opinion.

Reply:
> If an option is set globally as CLI argument and in the schema file, using the value specified in the schema file is more user-friendly in my opinion.

I agree. That's more user-friendly behavior.

Reply:
I added the options handling for the Schema Loader `--import` command in https://github.com/scalar-labs/scalardb/pull/1260/commits/b6e16cf10ca9b056f570d5062a9b5e8f53c1cc31 and it follows this principle.

> If an option is set globally as CLI argument and in the schema file, using the value specified in the schema file is more user-friendly in my opinion.

I will update the options handling for the other commands besides `--import` (all using the `TableSchema` class ) in another PR. 
Thanks for your comment and PTAL when you get the chance.